generation/copy_readme_changelog.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+for path in $(find . -mindepth 2 -maxdepth 2 -name pom.xml | sort | xargs dirname); do
+  if [[ "${path}" =~ google-cloud-gapic-bom ]] || [[ "${path}" =~ CoverageAggregator ]]; then
+    continue
+  fi
+  pushd "${path}"
+
+  module_name="${path:2}"
+  raw_readme_url="https://raw.githubusercontent.com/googleapis/${module_name}/main/README.md"
+  raw_changelog_url="https://raw.githubusercontent.com/googleapis/${module_name}/main/CHANGELOG.md"
+
+  curl -s "${raw_readme_url}" > README.md
```

Comment:
You should probably handle the case where the file is not found. It could happen when we only add the new library in the monorepo before the first release. That way you also wouldn't have to have the `continue` statement above.

Reply:
Yeah, that makes sense. I'll add the curl command to check for the URL.